scene/2d/tile_map.cpp

```diff
@@ -2668,19 +2668,25 @@ Rect2 TileMap::get_used_rect() { // Not const because of cache
 	// Return the rect of the currently used area
 	if (used_size_cache_dirty) {
 		bool first = true;
+		used_size_cache = Rect2();
```

Comment:
Should be `Rect2i()`

Reply:
But the variable is of type Rect2 and it's also assigned with Rect2 few lines below.

Reply:
Ah, I think that's a mistake I did then, I see no reason why the variable is Rect2 instead of Rect2i. 
Could you hange it?

Reply:
Renaming it to `used_rect_cache` would make sense too, it's not just size. And the same for `used_size_cache_dirty`.

Reply:
> Renaming it to used_rect_cache would make sense too, it's not just size. And the same for used_size_cache_dirty.

Ah yeah, that's true.